index.js

```diff
@@ -162,6 +163,19 @@ Consumer.prototype._processMessage = function (message, cb) {
       } else {
         consumer.emit('processing_error', err, message);
       }
+
+      if (consumer.terminateVisibilityTimeout) {
```

Comment:
We aim to maintain 100% test coverage, so we'll need some tests for this new functionality. To check test coverage run `npm run coverage`. It's also worth running the linter too. 

For more details see:
https://github.com/bbc/sqs-consumer/blob/master/CONTRIBUTING.md